ledger/ledger_test.go

```diff
@@ -2138,7 +2162,7 @@ func TestLedgerMigrateV6ShrinkDeltas(t *testing.T) {
 
 	_, err = l.OnlineTotals(basics.Round(proto.MaxBalLookback - shorterLookback))
 	require.Error(t, err)
-	for i := l.Latest() - basics.Round(proto.MaxBalLookback+shorterLookback-1); i <= l.Latest(); i++ {
```

Comment:
please explain

Reply:
Because the ledger was restarted, the acctonline lost all the uncommitted rounds and only holds the 320 rounds before l.latest()